code/modules/mob/living/basic/vermin/axolotl.dm

```diff
@@ -9,24 +9,33 @@
 	attack_verb_continuous = "nibbles" //their teeth are just for gripping food, not used for self defense nor even chewing
 	attack_verb_simple = "nibble"
 	butcher_results = list(/obj/item/food/nugget = 1)
+	density = FALSE
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	mob_size = MOB_SIZE_TINY
+	mob_biotypes = MOB_ORGANIC | MOB_BEAST
+	gold_core_spawnable = FRIENDLY_SPAWN
+
 	response_help_continuous = "pets"
 	response_help_simple = "pet"
 	response_disarm_continuous = "gently pushes aside"
 	response_disarm_simple = "gently push aside"
 	response_harm_continuous = "splats"
 	response_harm_simple = "splat"
-	density = FALSE
-	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
-	mob_size = MOB_SIZE_TINY
-	mob_biotypes = MOB_ORGANIC|MOB_BEAST
-	gold_core_spawnable = FRIENDLY_SPAWN
+
 	can_be_held = TRUE
 	held_w_class = WEIGHT_CLASS_TINY
-	worn_slot_flags = ITEM_SLOT_HEAD
 	held_lh = 'icons/mob/inhands/animal_item_lefthand.dmi'
 	held_rh = 'icons/mob/inhands/animal_item_righthand.dmi'
+	worn_slot_flags = ITEM_SLOT_HEAD
 	head_icon = 'icons/mob/clothing/head/pets_head.dmi'
 
-/mob/living/simple_animal/axolotl/Initialize(mapload)
+	ai_controller = /datum/ai_controller/basic_controller/axolotl
+
+/mob/living/basic/axolotl/Initialize(mapload)
```

Comment:
you need to make sure you add the components/elements that makes them breathe air and makes them be damaged by temperature :)

Reply:
Good catch! Did some testing on it and found out that none of the current basic mobs will die in space, except for stickmen. I'll fix it for axolotls tomorrow. It might be better to make that the default for basic mobs unless overridden, like it was with simple mobs. Or maybe it was left out for a reason